web_src/less/_base.less

```diff
@@ -2121,8 +2121,9 @@ a.ui.label:hover {
   font: 12px var(--fonts-monospace);
   white-space: pre-wrap;
   word-break: break-all;
-  overflow-wrap: break-word;
-  word-wrap: break-word;
+  overflow-wrap: anywhere;
+  /* stylelint-disable-next-line declaration-property-value-no-unknown */
+  word-wrap: anywhere; // Firefox respects word-wrap more than overflow-wrap
```

Comment:
```suggestion
  overflow-wrap: anywhere;
```

Reply:
This alone fixes it for me, no need for the deprecated alias:

<img width="1053" alt="image" src="https://user-images.githubusercontent.com/115237/222790425-004390b1-45e1-4c37-8518-8896d6a5c560.png">
<img width="1275" alt="image" src="https://user-images.githubusercontent.com/115237/222790516-6c5fa0f3-5164-468d-b855-c5550f69c418.png">

Reply:
I also think it's time to drop the legacy `word-wrap` if there is no other consideration. The only surprise is that Firefox respects the legacy style more than the modern one.

Reply:
Maybe the surprise is caused by: if the Firefox engine use aliasing, the code  `{ overflow-wrap;  word-wrap; }` would become  `{  overflow-wrap; overflow-wrap; }`, so the second takes affect. While in my mind I thought the engine should respect the modern one first, if there is no modern on, use the legacy one.

Since the legacy style has been removed, I think it's not worth to spend more time on it 😂 So far so good.